content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -169,26 +169,27 @@ supported path types:
-| Prefix | `/aaa/bbb`                      | `/aaa/bbb`                    | Yes                                |
-| Prefix | `/aaa/bbb/`                     | `/aaa/bbb`                    | Yes, ignores trailing slash        |
-| Prefix | `/aaa/bbb`                      | `/aaa/bbb/`                   | Yes,  matches trailing slash       |
-| Prefix | `/aaa/bbb`                      | `/aaa/bbb/ccc`                | Yes, matches subpath               |
-| Prefix | `/aaa/bbb`                      | `/aaa/bbbxyz`                 | No, does not match string prefix   |
-| Prefix | `/`, `/aaa`                     | `/aaa/ccc`                    | Yes, matches `/aaa` prefix         |
-| Prefix | `/`, `/aaa`, `/aaa/bbb`         | `/aaa/bbb`                    | Yes, matches `/aaa/bbb` prefix     |
-| Prefix | `/`, `/aaa`, `/aaa/bbb`         | `/ccc`                        | Yes, matches `/` prefix            |
-| Prefix | `/aaa`                          | `/ccc`                        | No, uses default backend           |
-| Mixed  | `/foo` (Prefix), `/foo` (Exact) | `/foo`                        | Yes, prefers Exact                 |
+| Kind   | Path(s)                         | Request path(s)               | Matches?                            |
+|--------|---------------------------------|-------------------------------|-------------------------------------|
+| Prefix | `/`                             | (all paths)                   | Yes                                 |
+| Exact  | `/foo`                          | `/foo`                        | Yes                                 |
+| Exact  | `/foo`                          | `/bar`                        | No                                  |
+| Exact  | `/foo`                          | `/foo/`                       | No                                  |
+| Exact  | `/foo/`                         | `/foo`                        | No                                  |
+| Prefix | `/foo`                          | `/foo`, `/foo/`               | Yes                                 |
+| Prefix | `/foo/`                         | `/foo`, `/foo/`               | Yes                                 |
+| Prefix | `/aaa/bb`                       | `/aaa/bbb`                    | No                                  |
+| Prefix | `/aaa/bbb`                      | `/aaa/bbb`                    | Yes                                 |
+| Prefix | `/aaa/bbb/`                     | `/aaa/bbb`                    | Yes, ignores trailing slash         |
+| Prefix | `/aaa/bbb`                      | `/aaa/bbb/`                   | Yes,  matches trailing slash        |
+| Prefix | `/aaa/bbb`                      | `/aaa/bbb/ccc`                | Yes, matches subpath                |
+| Prefix | `/aaa/bbb`                      | `/aaa/bbb/ccc/ddd`            | No, /aaa/bbb is not a prefix of /ddd|
```

Comment:
I don't think this is correct. `/aaa/bbb` _is_ a prefix match for request path `/aaa/bbb/ccc/ddd`.

Reply:
I believe if the request path is `/aaa/bbb/ccc/ddd` then **yes** it matches the prefix `/aaa/bbb` for the same reason why request path to `/aaa/bbb/ccc` matches with prefix `/aaa/bbb`

I'll do a test when I have cycles

Reply:
The actual behavior will depend on the ingress controller you select (watch out Rey in case you pick a noncompliant one)

Reply:
For your reference: we use nginx-ingress in our setup. 
Sorry, I'm not able to add a sample at this time.
We ended up using the prefix but used a regex to let the path to match all that comes after the prefix.

Reply:
Tested this with nginx-ingress
Requests to `/aaa/bbb/ccc/ddd` went to `/aaa/bbb`